test/unit/test_sap_cli__entry.py

```
@@ -303,6 +348,57 @@ def test_args_ssl_server_cert_default_none(self):
 
         self.assertIsNone(args.ssl_server_cert)
 
+    def test_args_auth_plugin_disable_cache_default_false(self):
+        """Without the flag and without env, the resolved value is False
+           - the cache is on by default (the on-disk store is opt-out)."""
+
+        test_params = get_tested_parameters()
+
+        with patch.dict(os.environ, {}, clear=False) as _env:
```

Flake8 (7.3.0)

[error] local variable '_env' is assigned to but never used

(F841)